scripts/ci/node-template-release/Cargo.toml

```diff
@@ -11,13 +11,11 @@ publish = false
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-clap = { version = "4.0.9", features = ["derive"] }
+clap = { version = "4.2.5", features = ["derive"] }
 flate2 = "1.0"
-fs_extra = "1"
-git2 = "0.8"
-glob = "0.2"
+fs_extra = "1.3"
+git2 = "0.16"
+glob = "0.3"
 tar = "0.4"
 tempfile = "3"
-toml = "0.4"
-
-[workspace]
```

Comment:
I am not sure about this line of change.  
It will probably break somewhere downstream? Need to notify there or wait for it to break :see_no_evil:

Reply:
No. This is a completely internal thing.